Makefile

```
@@ -72,25 +70,16 @@ build: $(BINS)
 $(BIN_DIR)/%-$(BIN_ARCH): $(SRC) go.mod go.sum
 	mkdir -p $(BIN_DIR)
 	$(GOBUILD) -ldflags="-s -w -X main.version=$(GIT_REF) -X main.gitSha=$(GIT_SHA)" \
-	-o "$@" \
+	-o "$(addsuffix $(BIN_RELEASE_SUFFIX),$@)" \
 	"./$(CMD_DIR)/$(*)"
 
-## Pack binaries with upx
-pack: $(PACKED_BINS)
-.PHONY: pack
-
-$(PACKED_DIR)/%-$(BIN_ARCH): $(BIN_DIR)/%-$(BIN_ARCH)
-	mkdir -p $(PACKED_DIR)
-	$(UPXCMD) --lzma --best -f -o "$@" "$<" \
-	&& touch "$@"
-
 ## Create release artifacts
 release-artifacts: $(RELEASE_ARTIFACTS)
 .PHONY: release-artifacts
 
-$(RELEASE_DIR)/%-$(RELEASE_SUFFIX): $(PACKED_DIR)/%-$(BIN_ARCH)
+$(RELEASE_DIR)/%-$(RELEASE_SUFFIX): $(BIN_DIR)/kubent-$(BIN_ARCH)$(BIN_RELEASE_SUFFIX)
 	mkdir -p $(RELEASE_DIR)
-	$(TAR) -cvz --transform 's,$(PACKED_DIR)/$(*)-$(BIN_ARCH),$(*)$(BIN_RELEASE_SUFFIX),gi' -f "$@" "$<"
+	$(TAR) -cvz --transform 's,$(BIN_DIR)/$(*)-$(BIN_ARCH)$(BIN_RELEASE_SUFFIX),$(BIN_DIR)/$(*)$(BIN_RELEASE_SUFFIX),gi' -f "$@" "$<"
```

```suggestion
	$(TAR) -cvz --transform 's,$(BIN_DIR)/$(*)-$(BIN_ARCH)$(BIN_RELEASE_SUFFIX),$(*)$(BIN_RELEASE_SUFFIX),gi' -f "$@" "$<"
```
This is why the builds currently fail - this change is adding `bin/` prefix compared to the current state.